packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -437,14 +439,32 @@ export function mapCustomChainToNetworkData(chain: ChainWithRpcUrl) {
   explorerUrls[chain.chainID] = chain.explorerUrl
 }
 
-function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
+export function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
   return !chain.isArbitrum
 }
 
 function isArbitrumChain(chain: L1Network | L2Network): chain is L2Network {
   return chain.isArbitrum
 }
 
+export const TELEPORT_ALLOWLIST: { [id: number]: number[] } = {
+  [ChainId.Ethereum]: [1380012617, 70700], // Rari and PopApex
+  [ChainId.Sepolia]: [ChainId.StylusTestnetV2]
+}
+
+export function addTeleportDestinationChainToList(
```

Comment:
I don't think we need this util

Reply:
We can have an util for this but I'd structure it something like:

```typescript
function getChildChainIds(chain: Chain) {
   const childChainIds = [...(chain.partnerChainIds ?? []), ...TELEPORT_ALLOWLIST[chain.chainId]]
   return Array.from(new Set(childChainIds))
}
```

Reply:
Good suggestion! Updated.